MobileFilter/sections/specific_web.txt

```diff
@@ -130,7 +129,7 @@ enfant.living.jp##.ad-inread
 arine.jp##.Entry-top-ad
 news.yahoo.co.jp##a[data-cl-params$="cl_link:bnr_b;"]
 m.malaysiakini.com##.items-center[style="height:300px"]
-kabumatome.doorblog.jp##.plugin-free_area
+vipsister23.com,kabumatome.doorblog.jp##.plugin-free_area
```

Comment:
Hm, apparently this is not ad leftover but pls check @zloyden

Reply:
Looks like this is not a leftover - I do not see ads without AdGuard and with Japanese IP

Reply:
ok, i removed it

Reply:
@SKEIDs in my view, we can leave this rule, please revert it